internal/session/repository_session.go

```diff
@@ -700,6 +701,20 @@ func (r *Repository) checkIfNoLongerActive(ctx context.Context, reportedSessions
 	return notActive, nil
 }
 
+func (r *Repository) deleteTerminated(ctx context.Context, threshold time.Duration) (int, error) {
```

Comment:
Could we rename this something more illustrative? How about `deleteTerminatedBefore`?

Reply:
Even `OlderThan` seems ambiguous or not clear to me. I am struggling to find a concise way of saying "delete sessions that have been in the terminated state for longer than the given threshold". I am open to ideas, but given that this is un-exported, it might be best to just add a comment?

Reply:
I think `deleteSessionsTerminatedBefore` might be the clearest I can think of.

Reply:
Sounds good